api/pkg/service/catalog/catalog.go

```diff
@@ -79,6 +79,35 @@ func (s *service) Refresh(ctx context.Context, p *catalog.RefreshPayload) (*cata
 	return ret, nil
 }
 
+// refresh all catalog for new resources
+func (s *service) RefreshAll(ctx context.Context, p *catalog.RefreshAllPayload) ([]*catalog.Job, error) {
+
+	log := s.Logger(ctx)
+	db := s.DB(ctx)
+
+	ctgs := []model.Catalog{}
+	if err := db.Model(&model.Catalog{}).Find(&ctgs).Error; err != nil {
```

Comment:
Can't we use directly `db.find` since we don't want any specific fields from the tables ?